README.md

```diff
@@ -1,24 +1,11 @@
 # DiscordSRV Documentation
 
-This is the documentation for DiscordSRV that uses [Mkdocs](https://github.com/mkdocs/mkdocs) to generate a static site hosted on https://docs.discordsrv.com.
+This is the documentation for DiscordSRV that uses [Docusaurus](https://docusaurus.io/) to generate a static site hosted on https://docs.discordsrv.com.
 
 ## Building
 
-[Create and activate a Python 3 virtual environment](https://docs.python.org/3/tutorial/venv.html)
-```sh
-$ pip install --user virtualenv
-$ virtualenv env
-$ source env/bin/activate
-```
+to install dependencies, run `yarn install`|`npm install`|`pnpm install`
 
-Install the required packages
-```sh
-$ pip install -r requirements.txt
-```
+to start a development server, run `yarn run start`|`npm run start`|`pnpm run start`
```

Comment:
The repo has npm & yarn lock files atm, should only have one or the other,

then the readme should only have instructions for that one dependency manager